mmcls/models/losses/cross_entropy_loss.py

```diff
@@ -18,15 +18,63 @@ def cross_entropy(pred, label, weight=None, reduction='mean', avg_factor=None):
     return loss
 
 
+def binary_cross_entropy(pred,
+                         label,
+                         weight=None,
+                         reduction='mean',
+                         avg_factor=None):
+    """Calculate the binary CrossEntropy loss with logits.
+
+    Args:
+        pred (torch.Tensor): The prediction with shape (N, C), where C is the
+            number of classes.
+        label (torch.Tensor): The learning label with shape (N, C), where C is
+            the number of classes.
```

Comment:
I thought it might be `(N, *)` instead of `(N, C)`, because it is for binary classification.  
FYI. https://pytorch.org/docs/stable/generated/torch.nn.BCELoss.html?highlight=bce#torch.nn.BCELoss